app/src/main/java/com/github/se/orator/ui/mainScreen/MainScreen.kt

```diff
@@ -107,11 +93,18 @@ fun ButtonRow(navigationActions: NavigationActions) {
  * The implementation of a button
  */
 @Composable
-fun SectionButton(text: String, onClick: () -> Unit) {
+fun SectionButton(text: String, onClick: () -> Unit, isSelected: Boolean) {
+  var selectedColor = MaterialTheme.colorScheme.secondary
+  var boldIfSelected = FontWeight.Normal
+  if (isSelected) {
+    selectedColor = MaterialTheme.colorScheme.primary
+    boldIfSelected = FontWeight.Bold
+  }
```

Comment:
Great job for the UI. You could rewrite this for more clarity:
```suggestion
fun SectionButton(text: String, onClick: () -> Unit, isSelected: Boolean) {
  val selectedColor = if (isSelected) MaterialTheme.colorScheme.primary else MaterialTheme.colorScheme.secondary
    val boldIfSelected = if (isSelected) FontWeight.Bold else FontWeight.Normal
}
```

Reply:
Thank you. Your way is better indeed.